tests/conftest.py

```diff
@@ -109,9 +109,9 @@ def test_with_any_microvm(test_microvm_any):
 # Tests root directory.
 SCRIPT_FOLDER = os.path.dirname(os.path.realpath(__file__))
 
-# This codebase uses Python features available in Python 3.6 or above
-if sys.version_info < (3, 6):
-    raise SystemError("This codebase requires Python 3.6 or above.")
+# This codebase uses Python features available in Python 3.10 or above
+if sys.version_info < (3, 10):
+    raise SystemError("This codebase requires Python 3.10 or above.")
```

Comment:
Can this be an assertion?
```suggestion
assert sys.version_info < (3, 10), "This codebase requires Python 3.10 or above."
```